### PR TITLE
VPN - documentation edited (note ???s)

### PR DESCRIPTION
**Pull request by alexander-levitski**

### CODE REVIEW

File: community/cloud-foundation/templates/vpn/vpn.py.schema [hunk 1]
```diff
@@ -27,47 +27,47 @@ required:
 properties:
   network:
     type: string
-    description: URI of the network to which this router belongs.
+    description: The URI of the network to which the router ??? VPN ??? belongs.
```
Comment (sourced-vince) [inline community/cloud-foundation/templates/vpn/vpn.py.schema]: VPN

File: community/cloud-foundation/templates/vpn/vpn.py.schema [hunk 2]
```diff
@@ -27,47 +27,47 @@ required:
...
   region:
     type: string
-    description: URI of the region where the router resides.
+    description: The URI of the region where the router ??? VPN ??? resides.
```
Comment (sourced-vince) [inline community/cloud-foundation/templates/vpn/vpn.py.schema]: VPN

File: community/cloud-foundation/templates/vpn/vpn.py.schema [hunk 3]
```diff
@@ -27,47 +27,47 @@ required:
...
-    description: IP address of the peer VPN gateway. Only IPv4 is supported.
+    description: The IP address of the peer VPN gateway. Only IPv4 is supported.
     pattern: ^([0-9]{1,3}\.){3}[0-9]{1,3}$
   asn:
     type: integer
     description: |
-      Local BGP Autonomous System Number (ASN). Must be an RFC6996 private ASN,
-      either 16-bit or 32-bit. The value will be fixed for this router
+      The local BGP Autonomous System Number (ASN). Must be an RFC6996 private ASN,
+      either 16-bit or 32-bit. The value will be fixed for this router ??? VPN ???
```
Comment (sourced-vince) [inline community/cloud-foundation/templates/vpn/vpn.py.schema]: VPN

File: community/cloud-foundation/templates/vpn/README.md [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
-Templated VPN deployment
+This template creates a VPN.
 
 ## Prerequisites
 
 - Install [gcloud](https://cloud.google.com/sdk)
-- Create a [GCP project, setup billing, enable requisite APIs](../project/README.md)
+- Create a [GCP project, set up billing, enable requisite APIs](../project/README.md)
 - Create a [network](../network/README.md)
+- ??? roles/permissions ???
```
Comment (sourced-vince) [inline community/cloud-foundation/templates/vpn/README.md]: - Grant the [compute.networkAdmin or compute.admin](https://cloud.google.com/compute/docs/access/iam) IAM role to the project service account

**Comment by alexander-levitski** [comment]
All done. Can you please approve? Or do we need a second reviewer/approver?

File: community/cloud-foundation/templates/vpn/vpn.py.schema [hunk 5]
```diff
@@ -39,8 +39,8 @@ properties:
     type: integer
     description: |
       The local BGP Autonomous System Number (ASN). Must be an RFC6996 private ASN,
-      either 16-bit or 32-bit. The value will be fixed for this router ??? VPN ???
-      resource. All VPN tunnels that link to this router will have the same
+      either 16-bit or 32-bit. The value will be fixed for the VPN
```
Comment (sourced-vince) [inline community/cloud-foundation/templates/vpn/vpn.py.schema]: All the documentation should be 80 chars length at most. I believe the word `resource could fit on this line.

**Comment by alexander-levitski** [comment]
Done

File: community/cloud-foundation/templates/vpn/tests/integration/vpn.bats [hunk 6]
```diff
@@ -34,11 +34,11 @@ function delete_config() {
 }
 
 function setup() {
-    # Global setup - this gets executed only once per test file
+    # Global setup; this is executed once per test file.
     if [ ${BATS_TEST_NUMBER} -eq 1 ]; then
         gcloud compute networks create "network-${RAND}" \
             --project "${CLOUD_FOUNDATION_PROJECT_ID}" \
-            --description "integration test ${RAND}" \
+            --description "Integration test ${RAND}" \
```
Comment (sourced-vince) [inline community/cloud-foundation/templates/vpn/tests/integration/vpn.bats]: Please revert. This will break the test.

**Comment by alexander-levitski** [comment]
Done. Approve?

**Pull request closed without merging**